subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy

```diff
@@ -80,6 +81,14 @@ class WrapperGenerationIntegrationTest extends AbstractIntegrationSpec {
         file("first/gradlew.bat").md5Hash == file("second/gradlew.bat").md5Hash
     }
 
+    def "can successfully run the wrapper task twice"() {
+        when:
+        succeeds("wrapper", "--gradle-version=5.0")
```

Comment:
Why are we passing the version here? It should fail as well when we create a wrapper for the current Gradle version, right?

Reply:
If I don't specify a version it'll create a URL that doesn't exist yet. I'll think of something else.

Reply:
If you use `AbstractWrapperIntegrationSpec`, you should be able to use `prepareWrapper` to generate a URL to a local bin distribution and the `wrapperExecutor` to run the build.

Reply:
Thanks, that's exactly what I need.